people.py

```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
     key = new_p.first_name + '_' + new_p.surname
     people[key] = new_p
 
+
+def _add_squanchy(people):
+    """ adds squanchy """
+    new_p = PERSON(first_name='squanchy', surname='smith',
+                   age=21, pet='Rat', hobbies=['squanching','running'])
```

Comment:
Need a white space after the comma `['squanching','running']`

Reply:
Responding to this PR review is TW5 exercise please see:

https://canvas.anglia.ac.uk/courses/14266/discussion_topics/132367

Reply:
fixed in f25f16f
is this ok?